### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wanda-carlson**
Add description

### CODE REVIEW

**Comment by wanda-carlson** [comment]
comment

**Comment by wanda-carlson** [comment]
**Chad Daniels** Hey can you view this?

**Comment by wanda-carlson** [comment]
old pr

**Comment by wanda-carlson** [comment]
old pr

**Comment by bert-griffin** [comment]
bert comment 2

**Comment by henry-richard** [comment]
henry comment

**Comment by wanda-carlson** [comment]
old pr

**Comment by henry-richard** [comment]
henry comment 2

**Comment by bert-griffin** [comment]
bert comment 2

**Comment by wanda-carlson** [comment]
comment

File: README.md [hunk 1]
```diff
@@ -3,3 +3,4 @@
 draft pr
 Adding suggestions
 test
+Test
```
Comment (wanda-carlson) [inline README.md:6]:
```suggestion
Testing
```

**Comment by wanda-carlson** [comment]
**Sarah Thompson** Can you check this out?

**Pull request merged**